### PR TITLE
Added figma info to documentation site

### PR DESCRIPTION
**Pull request by adam-parsons-ca**
Resolves issue #125. Please checkout if the copy and links are all working fine.

### CODE REVIEW

File: website/src/docs/1.0/design-system/figma-library.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+# Figma library
+
+The Figma library contains all the current components to help you design and build Citizens Advice digital products. Figma is an industry standard design software for UI design that can run in your browser.
+
+[Open Figma library ](https://www.figma.com/file/BXpZ8oSZHB9sGMPDwahFaJ/Citizens-Advice-design-system-library?node-id=0%3A404)
+
+## Using the Figma library
+
+Included in the library is a collection of predefined components and text styles for design and rapid prototyping. To get edit access to the library, you can [email the design team](mailto:design-team@citizensadvice.org.uk).
```
Comment (ChrisBrettCA) [inline website/src/docs/1.0/design-system/figma-library.md]: Does that email address exist already or something we'll get created?

**Comment by adam-parsons-ca** [comment]
Good point @ChrisBrettCA I need to remove it. It is contained in a "media" directory that has nothing else in it. Am I fine to delete the directory as well?

On another note, do we have the external link icon in the design system?

**Comment by ChrisBrettCA** [comment]
> Good point @ChrisBrettCA I need to remove it. It is contained in a "media" directory that has nothing else in it. Am I fine to delete the directory as well?
> 
> On another note, do we have the external link icon in the design system?

No I'm not sure we do, let's create a separate issue to track that.

**Comment by Lyndsey28** [comment]
@adam-parsons-ca should the figma file open in a new window

**Comment by Lyndsey28** [comment]
Figma library
The Figma library contains all the current components and text styles to help you design Citizens Advice digital products. Figma is an industry standard design software for UI design that can run in your browser.

View our Figma library

Using the Figma library
To use the Figma library components in your project email the [design team] for access.

**Comment by Lyndsey28** [comment]
@adam-parsons-ca 

Getting started
You can get started by installing the frontend framework or using the Figma library. 

-change link to the figma library to the documentation page

**Comment by adam-parsons-ca** [comment]
@ChrisBrettCA and @Lyndsey28  I have just made the change as per review yesterday. Please give them a once over so we can merge them in

**Pull request merged**